btc-node-helper/index.js

```diff
@@ -88,14 +88,22 @@ module.exports = class BtcNodeHelper {
         return promisefy(this.client, this.client.generate, [blocks]);
     };
     
-    getBlock(blockHash) {
-        return promisefy(this.client, this.client.getBlock, [blockHash]);
+    getBlock(blockHash, deserialized = true) {
+        return promisefy(this.client, this.client.getBlock, [blockHash, deserialized]);
+    }
+
+    getBlockCount() {
+        return promisefy(this.client, this.client.getBlockCount, []);
     }
 
     getUtxos(address) {
         return promisefy(this.client, this.client.listUnspent, [null, null, [address]]);
     }
 
+    execute(cmd, args = []) {
+        return promisefy(this.client, this.client.cmd, [cmd].concat(args));
```

Comment:
I'm a bit confused about this part `[cmd].concat(args)`

Why is it passing the cmd name as a parameter?

Reply:
Did you try this?

Reply:
This is simply sending the command and arguments as an array like: ['importaddress', 'address', 'label'] to the `this.client.cmd` function.